js/screens/ProfileScreen.js

```diff
@@ -49,7 +49,7 @@ class ProfileScreen extends React.Component<Props> {
 
 const mapStateToProps = (state: GlobalState): ReduxMappedProps => ({
   user: state.user,
-  navigation: state.navigation
+  navigation: state.navigation // FIXME: this is missing from ReduxMappedProps
```

Comment:
not sure this should be here since it's missing from `ReduxMappedProps`, is this needed at all? shouldn't this value be injected by `connect` ?

Reply:
You are right I am going to remove navigation from mapStateToProps. The reason is the `navigation` property is being passed from the Navigator to each screen. It is not related to `connect`.

Reply:
👍 let me know if you want to do that in this PR or in a different one (in that case I'll close this one)